helion/autotuner/base_search.py

```diff
@@ -409,7 +409,7 @@ def parallel_benchmark(
         iterator = iter_with_progress(
             zip(configs, fns, is_workings, strict=True),
             total=len(configs),
-            description=f"{desc}: exploring neighbors",
+            description=f"{desc} exploring neighbors",
```

Comment:
nit: curious if we intentionally want to remove `:`

Reply:
Yes, it was printing `"::"` since desc already had `:`